packages/http/src/getHttpOperations.ts

```diff
@@ -1,13 +1,16 @@
-import { transformOas2Operation, transformOas3Operation } from '@stoplight/http-spec';
+import { transformOas2Operations, transformOas3Operations, transformPostmanCollectionOperations } from '@stoplight/http-spec';
```

Comment:
@karol-maciaszek I think it's time to move this logic outside of the http package and put it in the CLI, this stuff is likely useless here.